core/block_svg.js

```diff
@@ -689,39 +694,46 @@ Blockly.BlockSvg.prototype.onMouseUp_ = function(e) {
     Blockly.Events.fire(
         new Blockly.Events.Ui(this, 'click', undefined, undefined));
   }
+  // This must be before terminateDrag, which deletes the manager.
+  var manager = Blockly.draggedConnectionManager_;
+  if (manager) {
+    var wasDragging = true;
+    var highlightedConnection = manager.closestConnection_;
```

Comment:
It seems a little weird to make a bunch of private variables on this new object then use them.